Refactor to move my code into it's own module.

ni3po42 opened the pull request:
This pull request is more or less a test on performing a pull request with the added benefit that I started updating my piece of this pie.
Aside from refactoring my piece into it's own module, I've commented out some offending bits that was impacting the build.
I may have also added eslint.

ni3po42 committed:
Refactor to move my code into it's own module.

ni3po42 committed:
Adding game of life (WIP)

ni3po42 committed:
Closer.. .but form is acting really bizare, gonna start unit tests

ni3po42 committed:
canvas renderer works now

tabrown3 commented:
Very cool, an implementation of Conway's Game of Life! And thanks for making the project compile again. I don't have time right at this moment, but later this evening I will pull down your code, see what it's about, and almost certainly accept the pull request so it's added to the repo.

ni3po42 committed:
More Game of life additions

Added text renderer and selector for renderers. not fully working yet, though

ni3po42 committed:
Fixed render selection and a host of other issues. no unit test :(

ni3po42 commented:
Ok. Now I'm done adding stuff for now.

ni3po42 committed:
fix build bug

weird error occurs while building on a different machine; required me to export const of my InjectionTokens themselves. I'd like to improve the way this work in that file , but for now it will build

ni3po42 committed:
Injection update and text render update

began working on text renderer that will alow multi-colors and run quick, right now neither is the case :(.
